package.json

```diff
@@ -12,7 +12,9 @@
     "deploy:cf": "yarn --cwd oneapp-server deploy:pre && cf push usds-nj-oneapp-staging-api --strategy rolling",
     "deploy:staging": "yarn deploy:cf"
   },
-  "dependencies": {},
+  "dependencies": {
+    "soap": "^0.35.0"
+  },
```

Comment:
We can remove the dependency since we're not going to integrate with the actual API for the MVP.